packages/cli/src/chains.ts

```diff
@@ -1,18 +1,7 @@
 import * as viem from 'viem';
 import * as viemChains from 'viem/chains';
 
-export const cannonChain: viem.Chain = {
-  id: 13370,
-  name: 'Cannon Local',
-  nativeCurrency: {
-    name: 'Ether',
-    symbol: 'ETH',
-    decimals: 18,
-  },
-  rpcUrls: { default: { http: ['http://localhost:8545'] } },
-};
-
-export const chains: viem.Chain[] = [cannonChain, ...Object.values(viemChains)];
+export const chains: viem.Chain[] = [...Object.values(viemChains)];
```

Comment:
Why this change?

Reply:
They added cannon to the viem list so it's redundant now